src/Integration.UnitTests/LocalServices/ErrorListInfoBarController.QualityProfileBackgroundProcessorTests.cs

```diff
@@ -100,21 +100,6 @@ public void QualityProfileBackgroundProcessor_QueueCheckIfUpdateIsRequired_ArgCh
             Exceptions.Expect<ArgumentNullException>(() => testSubject.QueueCheckIfUpdateIsRequired(null));
         }
 
-        [TestMethod]
-        public void QualityProfileBackgroundProcessor_QueueCheckIfUpdateIsRequired_NoFilteredProjects()
```

Comment:
This test and the next one seemed to be testing the same condition, so I removed this one.